dist/index.html

```diff
@@ -1,35 +1,29 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" dir="ltr">
   <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link
-      rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/fork-awesome@1.2.0/css/fork-awesome.min.css"
-      integrity="sha256-XoaMnoYC5TH6/+ihMEnospgm0J1PM/nioxbOUdnM8HY="
-      crossorigin="anonymous"
-    />
-    <title>ToDo List</title>
-    <script defer src="/runtime.bundle.js"></script>
-    <script defer src="/index.bundle.js"></script>
+    <meta charset="utf-8" />
+    <title>To-do List 2022</title>
+    <script defer src="bundle.js"></script>
   </head>
   <body>
-    <div class="container">
-      <div class="today-list">
-        <h2 class="title">Today's To Do</h2>
-        <button type="button" class="btn btn-refresh">
-          <i class="fa fa-refresh" aria-hidden="true"></i>
-        </button>
+    <div class="main-panel">
+      <div class="title">
+        <h1 class="list-title">Today's To Do</h1>
+        <img src="icons8-refresh-24.png" alt="refresher" id="refresher" />
       </div>
-      <div class="input-list">
-        <input class="input" type="text" placeholder="Add to your list..." />
-        <button type="button" class="btn btn-add">
-          <i class="fa fa-plus-square" aria-hidden="true"></i>
-        </button>
-      </div>
-      <ul class="list"></ul>
-      <button type="button" class="btn btn-clear">Clear all completed</button>
+      <button type="button" class="btn-refresh">
+        <i class="fa fa-refresh" aria-hidden="true"></i>
+      </button>
```

Comment:
Please place this button on the right corner of the heading instead if at the bottom of the heading: this makes it unfriendly.